dbms/src/Flash/Statistics/JoinImpl.cpp

```diff
@@ -61,9 +49,14 @@ void JoinStatistics::collectExtraRuntimeDetail()
             }
         }
     }
+
+    /// In TiFlash, a hash join's build side is finished before probe side starts,
+    /// so the join probe side's running time does not include hash table's build time,
+    /// when construct ExecutionSummaries, we need add the build cost to probe executor
+    base.execution_time_ns += join_build_base.execution_time_ns;
```

Comment:
This treatment is wrong
From https://github.com/pingcap/tiflash/pull/6995/files#diff-8afda9cf273246b0b526e551688e59abad0ac41f2803a82e1342eb15bf1c1a4bL88-L109, here also need to add the execute time of all join build streams that executed before.